chpl/chpl-service/src/main/java/gov/healthit/chpl/job/MeaningfulUseUploadJob.java

```diff
@@ -141,11 +133,16 @@ public void run() {
             LOGGER.error(msg);
             addJobMessage(msg);
             updateStatus(100, JobStatusType.Error);
+        } finally {
             try {
-                parser.close();
+                if (parser != null && !parser.isClosed()) {
```

Comment:
I thought we were going to use the "try with resources" thing. So lines 55/56 should be put into the `try (BuffererdReader = new...` format

Reply:
I accidentally modified this file.  This is something that Katy is working on.  I'm going to revert my changes.